### PR TITLE
feat(vercel): theme select for light/dark mode

### PR DESCRIPTION
**Pull request by uncenter**

### CODE REVIEW

File: styles/vercel/catppuccin.user.css [hunk 1]
```diff
@@ -1,17 +1,23 @@
 /* ==UserStyle==
 @name           Vercel Catppuccin
 @namespace      github.com/catppuccin/userstyles/styles/vercel
+@homepageURL 	  https://github.com/catppuccin/userstyles/tree/main/styles/vercel
```
Comment (ryanccn) [inline styles/vercel/catppuccin.user.css:4]: Mixed indentation here

**Review by ryanccn — CHANGES_REQUESTED** [review]

File: styles/vercel/catppuccin.user.css [hunk 2]
```diff
@@ -1,17 +1,23 @@
...
 @version        0.0.4
```
Comment (ryanccn) [inline styles/vercel/catppuccin.user.css:5]:
```suggestion
@version        0.0.5
```

**Review by ryanccn — APPROVED** [review]

File: styles/vercel/catppuccin.user.css [hunk 3]
```diff
@@ -1,17 +1,23 @@
 /* ==UserStyle==
 @name           Vercel Catppuccin
 @namespace      github.com/catppuccin/userstyles/styles/vercel
-@version        0.0.4
+@homepageURL 	https://github.com/catppuccin/userstyles/tree/main/styles/vercel
```
Comment (ryanccn) [inline styles/vercel/catppuccin.user.css:4]: Still using tabs

**Review by ryanccn — CHANGES_REQUESTED** [review]

File: styles/vercel/catppuccin.user.css (hunk 3 above)
Comment (uncenter) [reply styles/vercel/catppuccin.user.css:4]: wait what-

**Review by ryanccn — APPROVED** [review]

**Pull request merged**